server/models/db/student.js

```diff
@@ -10,7 +10,21 @@ const Tables = require('../../constants/tables.json');
 async function getStudent(email, password) {
     return userModel.getUser(email, password, Tables.student, Student);
 }
+// This method inserts a student account given specific information.
+async function insertStudentAccount(firstName, lastName, type, studentID, email, password, phone, school_id ) {
+    //Data recieved from the front end sign up form
+ //Insert one row into a table
+     return db.run(`insert into ${Tables.student}(firstName, lastName, type, studentID, email, password, phone, school_id) values 
```

Comment:
Hi Nancy,
Please refer to the `005.do.create-worker-table.sql` for the proper naming of the table columns. For example, the attribute in the sql table is `student_id` and not `studentID` so this may not be inserting correctly. I would suggest adjusting the naming to ensure that it inserts properly.

Reply:
Hi Amy,
Thank you for suggesting that!
My action: I've changed the following discrepancies in naming of the table columns: student_id, school_id, first_name, and last_name. These are now in sync with the names in `004.do.create-student-table.sql.`

You can now see these changes the following locations:

line 14 of server/models/db>student.js
line 17 of server/models/db>student.js
line 18 of server/models/db>student.js

Please confirm if this takes care of your suggestion!

Reply:
Hi Nancy,
That's perfect. Thank you!